### PR TITLE
Chore/GH Actions on master

### PR DESCRIPTION
**Pull request by stephengtuggy**
Thank you for submitting a pull request and becoming a contributor to the Vega Strike Core Engine.

Please answer the following:

Code Changes:
- [ ] Have the PR Validation Tests been run? See https://github.com/vegastrike/Vega-Strike-Engine-Source/wiki/Pull-Request-Validation
- [ ] This is a documentation change only
- [x] This is a CI/CD change only

Issues:
- Please list any related issues
#382 

Purpose:
- What is this pull request trying to do? Move from Travis CI to GitHub Actions on master
- What release is this for? 0.8.0
- Is there a project or milestone we should apply this to? 0.8.x

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: 'GH Actions CI'
+
+on:
+  push:
+    branches: ['*master', '*0.[0-9].x']
```
Comment (BenjamenMeyer) [inline .github/workflows/ci.yml:5]: This might need to be  `[0-9]?` for when we get to multiple digits

**Review by BenjamenMeyer — APPROVED** [review]
Lgfm

Might have a slight approval

File: .github/workflows/ci.yml (hunk 1 above)
Comment (stephengtuggy) [reply .github/workflows/ci.yml:5]: Oh. True.

Comment (stephengtuggy) [reply .github/workflows/ci.yml]: Done

**Comment by stephengtuggy** [comment]
@BenjamenMeyer if you're satisfied, can you go ahead and merge this? I can't merge it, because it's waiting for the nonexistent Travis CI status to pass.

Thanks!

**Pull request merged**